frontend/lib/i18n/locales/nl/polling_station.json

```diff
@@ -8,8 +8,11 @@
   },
   "message": {
     "no_polling_stations": "Er zijn nog geen stembureaus ingevoerd voor deze verkiezing. Kies hoe je stembureaus gaat toevoegen.",
-    "polling_station_updated": "Wijzigingen stembureau {number} ({name}) opgeslagen.",
-    "polling_station_created": "Stembureau {number} ({name}) toegevoegd."
+    "polling_station_updated": "Wijzigingen stembureau {number} ({name}) opgeslagen",
+    "polling_station_created": "Stembureau {number} ({name}) toegevoegd",
+    "polling_station_deleted": "Stembureau {pollingStation} verwijderd",
```

Comment:
Is pollingStation here a number like in the sentences above or something else?

Reply:
It is the `{number} ({name})` combined, because for deleted, the polling station cannot be looked up in the list of polling stations anymore so the whole string is supplied to show in the message.